### ISSUE TITLE
Remove Accessible Icon from the flag icon in the RemainingFlagsCounter component.

### ISSUE THREAD

**Issue by v-gajjar** [v-gajjar/Minesweeper#53]
I was reading through this [article](https://lucide.dev/guide/advanced/accessibility) on icon accessibility and appear to have incorrectly understood how and when [Accessible Icon](https://www.radix-ui.com/primitives/docs/utilities/accessible-icon) should be used/best practises in terms of icon accessibility. More research is needed to better understand best practises regarding icon accessibility.
